### PR TITLE
README: SQLite specify encryption limitations

### PR DESCRIPTION
**Pull request by moio**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,6 +61,14 @@ Steve is used with SQLite caching of resources, which is configured when
 calling `server.New` via the `server.Options.SQLCache` boolean option.
 Meaning and behavior are the same unless otherwise specified.
 
+Note that, if SQLite caching of resources is enabled, some of the data
+is encrypted at rest. In particular:
```
Comment (pjbgf) [inline README.md:65]:
```suggestion
Note that, if SQLite caching of resources is enabled, some of the data
can be stored in disk, in either encrypted or plain text form based on:
```

File: README.md [hunk 2]
```diff
@@ -61,6 +61,14 @@ Steve is used with SQLite caching of resources, which is configured when
...
+ - by default, Secrets are encrypted
```
Comment (pjbgf) [inline README.md:66]:
```suggestion
 - by default, Secrets are always encrypted
```

File: README.md [hunk 3]
```diff
@@ -61,6 +61,14 @@ Steve is used with SQLite caching of resources, which is configured when
...
+ - if the environment variable `CATTLE_ENCRYPT_CACHE_ALL` is set to "true",
+all resources are encrypted
+ - regardless of the setting, any filterable/sortable columns are stored
```
Comment (pjbgf) [inline README.md:69]:
```suggestion
 - regardless of the setting value , any filterable/sortable columns are stored
```

**Review by ericpromislow — APPROVED** [review]
lgtm

**Review by pjbgf — APPROVED** [review]

**Review by ericpromislow — APPROVED** [review]
still lgtm

**Pull request merged**